wp-content/themes/goonj-crm/functions.php

```diff
@@ -236,6 +236,17 @@ function goonj_handle_user_identification_form() {
 			exit;
 		}
 
+		if ( 'office-visit-material-contribution' === $purpose ) {
```

Comment:
Aren't office visit and material contribution two different purpose?

I was expecting something like: `processing-center-material-contribution` and `processing-center-office-visit`

Reply:
sure updated